osbs/cli/main.py

```diff
@@ -44,6 +44,31 @@ def print_json_nicely(decoded_json):
     print(json.dumps(decoded_json, indent=2))
 
 
+def check_provided_args(args, check, check_func, error_msg):
+    """Checks arguments against a check function
+
+    :param args: argparse.Namespace, all provided arguments
+    :param check: list<str>, each argument to check
+    :param check_func: filter function for values
```

Comment:
Can this say 'items' instead of values, to make it clear why `[-1]` is necessary later?